<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEventDialog</class>
 <widget class="QDialog" name="CreateNewEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Event Name</string>
       </property>
       <property name="buddy">
        <cstring>eventNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eventNameLineEdit">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDateTimeEdit" name="startDateTimeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="endDateTimeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="allDayCheckBox">
       <property name="text">
        <string>All Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="repeatCheckBox">
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="eventDetailsGroupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>250</height>
      </size>
     </property>
     <property name="title">
      <string>Event Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Location</string>
        </property>
        <property name="buddy">
         <cstring>locationLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="locationLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Description</string>
        </property>
        <property name="buddy">
         <cstring>descriptionTextEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="buddy">
         <cstring>colorComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="colorComboBox">
        <item>
         <property name="text">
          <string>white</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lightGray</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QTextEdit" name="descriptionTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="repetitionGroupBox">
     <property name="title">
      <string>Repetition</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Repeat</string>
        </property>
        <property name="buddy">
         <cstring>repeatComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="repeatComboBox">
        <item>
         <property name="text">
          <string>Every Day</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every Week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every Month</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every Year</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Frequency</string>
        </property>
        <property name="buddy">
         <cstring>frequencyComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Start Date</string>
        </property>
        <property name="buddy">
         <cstring>repeatStartDateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDateEdit" name="repeatStartDateEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="repeatTimesRadioButton">
             <property name="text">
              <string>After Repeating</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="repeatTimesSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>3650</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Times</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="repeatEndTimeRadioButton">
             <property name="text">
              <string>End Date: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="repeatEndDateEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="repeatEndDateLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Summary</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLabel" name="repeatSummaryLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="frequencyComboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="repeatTimeLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startDateTimeEdit</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>repeatStartDateEdit</receiver>
   <slot>setDateTime(QDateTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditEventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>650</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
